media-sound/qpwgraph/qpwgraph-0.2.2.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg cmake
+
+DESCRIPTION="Qt GUI to control the JACK Audio Connection Kit and ALSA sequencer connections"
+HOMEPAGE="https://gitlab.freedesktop.org/rncbc/qpwgraph"
+SRC_URI="https://gitlab.freedesktop.org/rncbc/qpwgraph/-/archive/v${PV}/${PN}-v${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="alsa debug systray"
+
+BDEPEND="virtual/pkgconfig"
```

Comment:
In most ebuilds, `BDEPEND` is defined after `DEPEND` and `RDEPEND`. I suggest you stick with this convention.